src/workingtitle-vcockpits-instruments-airliners/html_ui/Pages/VCockpit/Instruments/Airliners/Shared/BaseAirliners.js

```diff
@@ -0,0 +1,2014 @@
+class BaseAirliners extends NavSystem {
```

Comment:
What change do we do here? 
Can we pull it into a WT folder and reference that as not to possibly overwrite others?

Reply:
I mentioned this file previously. I don't want to add it to our changed files since its a core file, but I can't isolate the code I added to it into a separate file. I need someone to help me do this.